src/utils/setup/create-community-champion-profile-pages.js

```diff
@@ -0,0 +1,34 @@
+import path from 'path';
+import { communityChampions } from '../../queries/community-champions';
+
+const createCommunityChampionProfilePage = async (champion, createPage) => {
+    const { firstName, lastName } = champion;
+    const slug = `/community-champions/${firstName.toLowerCase()}-${lastName.toLowerCase()}`;
+    createPage({
+        path: slug,
+        component: path.resolve(
+            `./src/templates/community-champion-profile.js`
+        ),
+        context: {
+            champion,
+            slug,
+        },
+    });
+};
+
+const getChampionListFromGraphql = async graphql => {
+    const championResp = await graphql(communityChampions);
+    const result = championResp.data.allStrapiCommunityChampions.nodes;
+    return result;
+};
+
+export const createCommunityChampionProfilePages = async (
+    createPage,
+    graphql
+) => {
+    const championList = await getChampionListFromGraphql(graphql);
+    championList.forEach(
+        async champion =>
+            await createCommunityChampionProfilePage(champion, createPage)
+    );
```

Comment:
Nice use of `async/await`! However I don't see us using any `await` in `createCommunityChampionProfilePage` so we likely don't need to `asnyc/await` at all here. I did see the same mistake for academia projects so I can see where maybe that came from :)

Reply:
are async functions useless if there are no `await`s in them?

Reply:
Yep, `asnyc` indicates that this function does something like a network call or so which should be handled properly, either by making the program `await` it or by using the JS Promise API. Even if nothing inside is actually asynchronous, it adds a bit of complexity when reasoning about the code, so if we don't need it we should avoid it!